README.md

```diff
@@ -209,11 +210,25 @@ export default new Vuex.Store({
 })
 ```
 
-#### Namespaced vuex modules
+#### Emitting Events  
+
+Events can be sent to the Socket.IO server by calling 
+```js
+this._vm.$socket.client.emit
+```
+from a Vuex mutation or action. Mutation or action names are not subject to the same naming requirements as above. Event data must be sent in string format.
```

Comment:
Event data can be send in any format (not just string)
https://socket.io/docs/client-api/#socket-emit-eventName-%E2%80%A6args-ack

Reply:
Good to know, thanks!